### DISCUSSION TITLE
Не работает смарт тв

### DISCUSSION THREAD

**Discussion by jura1243** [hufrea/byedpi#86] [General]
Настроил на сервере убунту запуск ./ciadpi --port 1080 --disorder 1 --fake 0 --ttl 1 -A torst --tlsrec 1+s --debug 0 &
Потом на роутере keenetic прокси сокс объявил общим интерфейсом, 
Добавил Dns over https от яндекса -
Все компы работают и ноут по вайфаю, а смарт тв нет. думал устаревшие тв (подключены по проводу). установил Xiaomi Mi TV Stick 4K EU (через вайфай) тоже самое. на ютюбе пишет нет коннекта, а все другие проги работают.
Почему на смарт тв ютюб не работает?

**Comment by ebkjiud** [comment]
предполагаю из-за quic протокола, которым пытается подключится

**Comment by ebkjiud** [comment]
![123](https://github.com/user-attachments/assets/7ea8f533-89c8-4c26-9f06-59cbd4716dbf)

**Comment by acidroper** [comment]
Рабочее решение для Keenetic и Android TV: https://github.com/Anonym-tsk/nfqws-keenetic

**Comment by 6apakyda** [comment]
Аналогичная ситуация. Только какие команды запуска не использовал, результат один и тот же, на ТВ где есть приложение ютуба, оно просто не открывается. Если исключить домен youtube.com из списка сайтов где используются фейковые пакеты, то приложение на ТВ открывается со скрипом, со скрипом подгружается список видео и при этом видео работают нормально. Подскажите направление (кроме VPN) как можно с помощью byedpi запускать приложение на ТВ без скрипа. Спасибо!
![Screenshot_2](https://github.com/user-attachments/assets/57cc5394-259b-4927-ae6e-8ce3a63ce00f)
![image](https://github.com/user-attachments/assets/4844c1ed-a661-4a3e-b059-5257c16d9c19)

**Comment by ebkjiud** [comment]
В zaprete вроде есть quic поддержка, но там иной механизм - зарулить в сокс
вроде для quic пока не находил метода

чт, 29 авг. 2024 г., 17:39 6apakyda ***@***.***>:

**Comment by jura1243** [comment]
Успешно запустил версию 0.16.4 на ТВ с помощью программы super proxy голубого цвета с двумя стрелочками.
Нужно чтоб на компе был запущен byedpi, а с помощью super proxy к нему коннектитесь. После подключения можно запускать ютюб приложение.
